app/models/druid_version_zip.rb

```diff
@@ -96,7 +111,7 @@ def work_dir
   # @see #work_dir
   # @return [String] shell command to create this zip
   def zip_command
-    "zip -vr0X -s 10g #{file_path} #{druid.id}/#{v_version}"
+    "zip -vr0X -sv -s #{zip_split_size} #{file_path} #{druid.id}/#{v_version}"
```

Comment:
stupidly naive question:  I assume if the file is < 10G, it will be a single file as before?

Reply:
Yes, that is default zip behavior.